### PR TITLE
added an event listener to copy username to chat input on double click

### PR DESCRIPTION
**Pull request by mattroseman**
This seemed like an intuitive UI thing that I would expect to happen. It is already a thing on twitch.

I'd make a PR to the destinygg/chat-gui repo but I think that's no longer maintained.

### CODE REVIEW

**Comment by vyneer** [comment]
Hey there! This is a really nice addition, I'd like for it to have a setting though, same as with other stuff (added it in a commit above!).

Also, not exactly sure how to do this, but if there's a way to change the base branch to "dev" instead of "main", that would be great!

By the way, I think chat-gui is still maintained, you just gotta poke some people in chat sometimes Hhhehehehe (Thanks for fixing that Android bug, I'm sure it'll get merged <3)

**Comment by mattroseman** [comment]
OK I merged the dev branch changes into mine, and redirected this PR. I think it picked up an update README commit you had made, so I can drop that if you don't want it.

Otherwise I tested with tampermonkey and I think it's gtg.

**Comment by vyneer** [comment]
Don't worry about the readme thing, prob good that it finally got synced up Hhhehehehe

Looks good to me, merging! <3

**Comment by Voiture-0** [comment]
If you enable this feature, and then refresh it won't work until you toggle the setting off and on again to re-add the event listener.  Outside of the checkbox listener a check needs to be added if the saved preference is true, to add the double click event listener. @mattroseman

File: dgg-utilities.user.js [hunk 1]
```diff
@@ -285,6 +290,53 @@ function injectScript() {
...
+doubleClickCopyCheck.addEventListener("change", () => {
+  doubleClickCopy = doubleClickCopyCheck.checked;
+  window.localStorage.setItem(
+    "vyneer-util.doubleClickCopy",
+    doubleClickCopyCheck.checked
+  );
+  window.removeEventListener('dblclick', doubleClickCopyListener);
+  if (doubleClickCopy) {
+    // if a username is double clicked copy it to the chat input
+    window.addEventListener('dblclick', doubleClickCopyListener);
```
Comment (Voiture-0) [inline dgg-utilities.user.js:334]: If you enable this feature, and then refresh it won't work until you toggle the setting off and on again to re-add the event listener. Outside of the checkbox listener a check needs to be added if the saved preference is true, to add the double click event listener.

Comment (vyneer) [reply dgg-utilities.user.js:334]: ahh, that's my bad, kinda rushed adding the settings checkbox Hhhehehehe
ill fix it! later MMMM

Comment (vyneer) [reply dgg-utilities.user.js:334]: i think i fixed it PepoG

**Pull request merged**